translations/kubectl/de_DE/LC_MESSAGES/k8s.po

```diff
@@ -1,6 +1,6 @@
 # German translation.
 # Copyright (C) 2017
-# This file is distributed under the same license as the PACKAGE package.
+# This file is distributed under the same license as the Kubernetes package.
```

Comment:
> Can these be corrected to refer to Kubernetes specifically?

@swinslow to double check, did you mean changing the literal `PACKAGE` to literal `Kubernetes` or something else?

Reply:
Hi @nikhita, yes, that's correct. Changing to `Kubernetes` is good here to avoid the ambiguity of `PACKAGE`.

Reply:
Thanks for the confirmation, @swinslow.